package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-connect-streams",
-  "version": "1.5.1",
+  "version": "1.5.2",
```

Comment:
Is this following semver? I think this prob needs to be a minor version update if it's adding new functionality and there are no breaking changes.